pyndl/preprocess.py

```diff
@@ -141,8 +142,9 @@ def process_occurrences(occurrences, outfile, *,
 
 def create_event_file(corpus_file,
                       event_file,
-                      symbols="abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ",
                       *,
+                      valid_symbols="*",
```

Comment:
I strongly opt against adding default characters to avoid any confusion. imho explicit is better than implicit in this case.
a-zA-Z can (and is) suggested in the docs.